Assets/Scripts/Login/Networking/LoginServerClient.cs

```diff
@@ -0,0 +1,271 @@
+            ));
+        }
+
+        /// <summary>
+        /// CentCom call to delete a character of the current user.
+        /// Requires Authenticate to have been called at least once before.
+        /// </summary>
+        /// <param name="id"></param>
+        /// <param name="resultCallback"></param>
+        public void DeleteCharacter(string id, Action<string, bool> resultCallback)
+        {
+            ValidateToken("Delete Character");
+            monoBehaviour.StartCoroutine(HandleSimpleRequest(
+                UnityWebRequest.kHttpVerbDELETE,
+                connectionAddress + Delete_DeleteCharacter + $"/{id}",
+                "Delete Character", 
+                true,
+                resultCallback
+            ));
+        }
+
+        /// <summary>
+        /// Handles requests that do not contain a data payload
+        /// </summary>
+        /// <param name="method">A value from UnityWebRequest.kHttpVerb<...></param>
+        /// <param name="url">The endpoint url where the request will be sent</param>
+        /// <param name="name">Name of this request. For logging purposes only.</param>
+        /// <param name="authenticatedRequest">Set to true, if the request requires authentication</param>
+        /// <param name="resultCallback">Callback to be executed when a response is received</param>
+        private IEnumerator HandleSimpleRequest(string method, string url, string name, bool authenticatedRequest, Action<string, bool> resultCallback)
+        {
+            Func<string, UnityWebRequest> requestCall;
+
+            switch (method)
+            {
+                case UnityWebRequest.kHttpVerbGET:
+                    requestCall = UnityWebRequest.Get;
+                    break;
+                case UnityWebRequest.kHttpVerbDELETE:
+                    requestCall = UnityWebRequest.Delete;
+                    break;
+                default:
+                    Debug.LogError($"Unknown request method {method}");
+                    yield break;
+            }
+            
+            using (UnityWebRequest webRequest = requestCall(url))
+            {
+                webRequest.SetRequestHeader("Content-Type", "application/json");
+                if (authenticatedRequest)
+                {
+                    webRequest.SetRequestHeader("Authorization", $"Bearer {token}");
+                }
+                yield return webRequest.SendWebRequest();
+                if (webRequest.isNetworkError || webRequest.isHttpError)
+                {
+                    Debug.Log($"{name} request error: {webRequest.error}");
+                    resultCallback(webRequest.downloadHandler.text, false);
+                    yield break;
+                }
+
+                string result = "";
+                if (webRequest.downloadHandler != null)
+                {
+                    result = webRequest.downloadHandler.text;
+                }
+                resultCallback(result, true);
```

Comment:
This is purely opinion, but I prefer inlining stuff like this to `resultCallback(webRequest.downloadHandler?.text, true);`
It will however return null instead of an empty string

Reply:
Should be able to do: ```webRequest.downloadHandler?.text ?? ""```, or worst case just a ternary operator.